ldap_probe/models.py

```diff
@@ -358,6 +362,54 @@ def node(self):
 
         return self.ad_node.get_node()
 
+    @property
+    def perf_alert(self):
+        """
+        flag for considering if an instance of this class must trigger a
+        performance alert
+
+        :returns: `True/False`
+        :rtype: bool
+        """
+        return any(
+            [
+                elapsed for elapsed in
+                [
+                    self_elapsed for self_elapsed in
+                    [
+                        self.elapsed_bind, self.elapsed_anon_bind,
+                        self.elapsed_search_ext, self.elapsed_read_root
+                    ]
+                    if self_elapsed is not None
```

Comment:
Instead of cascading list comprehension, just expand the condition
```suggestion
                    if self_elapsed is not None and self_elapsed >= get_preference('ldapprobe__ldap_perf_alert')
```

Reply:
i think you are missing the point of this property. it must return True if any of the elapsed_foo fields is both not None and gte than the threshold.
if you were to rewrite this without comprehensions, it would be a with a couple of ```if```s inside a ```for```.
Using ```any()``` with a list comprehension is the more pythonic way.